### DISCUSSION TITLE
Velero backing up Specific PVC FAILS - Ref : Restore a specific PVC by name #2043

### DISCUSSION THREAD

**Discussion by firstcommi** [vmware-tanzu/velero#7335] [Community support Q&A]
Hello !

This command ran succesfully , velero restore create nginxrestorepvc --from-backup daily-backup --include-resources pvc/nginx-pvc-trial --wait

but nginx-pvc-trial  has not been restored in my namespace

Note: I have taken all resources (My namespace)backed up in **daily-backup**

**Comment by draghuram** [comment]
You can only select resource types with this option. E,g, "pvcs". Selecting a specific PVC is not supported.

**Reply by firstcommi** [reply]
I was able to execute this successfully ,

" velero restore create mytestdeployrestore --from-backup daily-backup --include-resources deployments/my-testdeploy"

Selecting a specific PVC is not supported -can you brief the reason, please!

**Reply by firstcommi** [reply]
I have deleted one of the pvcs and tried to restore all.One that pvc of my namespace is not being restored and remains in a pending state due to some issue with the PV.Is it necessary to append cluster resources, even though the requirement is to take a backup of the PVCs within the namespace?"

Error : pv already bounded to a different claim

**Comment by draghuram** [comment]
Are you sure? As far as I know, Velero doesn't support restore of specific resources by name. May be you only have one deployment and it got restored. I will check code to verify the behavior.

**Comment by draghuram** [comment]
See https://kubernetes.slack.com/archives/C6VCGP4MT/p1706150444624129 for a related discussion.

**Reply by firstcommi** [reply]
This links is not accessible!

**Comment by shubham-pampattiwar** [comment]
+1 to what @draghuram has explained here.
